### DISCUSSION TITLE
ot-br and rdisc6 (ICMPv6 Router Discovery)

### DISCUSSION THREAD

**Discussion by rretanubun** [openthread/openthread#7641] [Q&A] [answered]
Hello,

I have a thread border router that is connected to a `LAN via ethernet -> [LAN switch] -> a linux PC` on the same IPv6 network
The linux PC can ping the unique local address of the border router (e.g. `fd06:64f8:de83:0000:7ae3:6dff:fe61:f53b`)

I then ran `rdisc6 eth1`  (eth1 being the linux network interface) and got a response for the [LAN switch], but not from the thread border router.

Is this a supported use case (being able to discover thread border router using ICMPv6 Router Discovery protocol) ? I saw `OT_ICMP6_TYPE_ROUTER_ADVERT and SOLICIT` in the thread source code so I thought it supports sending router advertisement , but I am clearly missing something.

Thank you for everyone's time!

**Comment by jwhui** [comment]
The default OpenThread Border Router build should generate ICMPv6 Router Advertisements and respond to ICMPv6 Router Solicitation messages. Can you provide more detail on how you configured and built your OpenThread Border Router and what hardware platform you are using?

**Comment by rretanubun** [comment, marked as answer]
Confirming Jonathan's answer, OpenThread Border Router does process `router solicitation` and respond with `router advertisement`. The issue for my setup is that the IP-driver layer (i.e. lwip) by default only consider well known IPv6 multicast for all IPv6 devices (`ff02::1`) for processing and `rdisc6` by default uses the all IPv6 router (`ff02::2`) which gets dropped at the IP layer.
